04_HTML/praktikum/createAccount.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <style>
+      label {
+        display: block;
+      }
+    </style>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Create Account</title>
+  </head>
+
+  <body>
+    <form action="/submit" method="post">
+      <h1>Create Account</h1>
+      <h2>Detail Form</h2>
+
+      <div>
+        <p>First name :</p>
+        <input type="text" />
+      </div>
+
+      <div>
+        <p>Last name :</p>
+        <input type="text" />
+      </div>
+
+      <div>
+        <p>Username :</p>
+        <input type="text" />
+      </div>
+
+      <div>
+        <p>Email :</p>
+        <input type="email" />
+      </div>
+
+      <div>
+        <p>Gender :</p>
+        <label><input type="radio" name="gender" />Male</label>
```

Comment:
kurang attribute for yg mana harus sama dg attribute id pada input element yg dituju